the-telephone-app-fe/src/components/DescribePicture/DescribePicture.js

```diff
@@ -32,18 +32,50 @@ const DescribePicture = () => {
     };
   }, [timer]);
 
-  const [content, setContent] = useState("");
+  let [content, setContent] = useState("");
 
   const handleChangeContent = (event) => {
     setContent(event.target.value);
   };
 
   const handleDone = async () => {
+    content = content.replace(new RegExp(" ", "g"), "_");
```

Comment:
could use another variable here to send the request instead of modify the content state.
let <variableName> = content.replace(new RegExp(" ", "g"), "_");